func_adl_xAOD/atlas/xaod/event_collections.py

```diff
@@ -0,0 +1,80 @@
+import func_adl_xAOD.common.cpp_ast as cpp_ast
+import func_adl_xAOD.common.cpp_types as ctyp
+
+from func_adl_xAOD.common.event_collections import event_collection_collection, event_collection_container, event_collections
+
```

Comment:
Use filenaming convention simlar to what you used in the cms (or normalize them, at least).